python_scripts/03_categorical_pipeline_column_transformer.py

```diff
@@ -82,10 +82,10 @@
 #   information will be correlated;
 # * **numerical scaling** numerical features which will be standardized.
 #
-# We can now create our `ColumnTransfomer` by specifying a list of triplet
-# (preprocessor name, transformer, columns). First, let's start by creating
-# a transformer for the numerical and categorical part as we did in the
-# previous notebooks.
+# We can now create our `ColumnTransfomer` by specifying three values:
```

Comment:
Yeah this is a bit weird here we are not creating the `ColumnTransformer` yet maybe we should say that we are creating the preprocessors for each type of column.

Then in the next cell, we say that the `ColumnTransformer` takes a list of (preprocessor_name, transformer, columns) or something like this.